src/dao/pages/DashboardHeader.tsx

```diff
@@ -5,22 +5,27 @@ import { DAOType } from "@dao/services";
 import { MintNFTModal } from "./MintNFTModal";
 import { useToken } from "@dex/hooks";
 import { Routes } from "@dao/routes";
+import { useContract } from "@dao/hooks";
 
 interface DashboardHeaderProps {
   isMember?: boolean;
   isAdmin?: boolean;
-  daoAccountId: string;
+  accountId: string;
   govTokenId?: string;
-  safeId?: string;
+  safeEVMAddress?: string;
   name: string;
   type: DAOType;
   logoUrl?: string;
   handleMintNFT?: (tokenLinks: string[]) => void;
 }
 
 export function DashboardHeader(props: DashboardHeaderProps) {
-  const { daoAccountId, name, type, logoUrl, govTokenId, safeId, isMember, isAdmin, handleMintNFT } = props;
+  const { accountId, name, type, logoUrl, govTokenId, safeEVMAddress, isMember, isAdmin, handleMintNFT } = props;
```

Comment:
If `accountId` is an EVM address and no longer an Account Id, appropriately rename the variable to explicitly highlight this change.

Reply:
accountId is the EVMAddress for MultiSig but for FT and NFT DAOs it is AccountId. so with this PR I have not changed the name of the variable. I wull update it with my next PR, in which I am making changes in the FT and NFT DAOs.

Reply:
@anshulkhare-lab49 Is this work documented in Rally? If not, create a new ticket or add this work to the requirements of an existing ticket.